.github/workflows/push.yml

```
@@ -18,3 +18,12 @@ jobs:
     with:
       build_type: branch
     secrets: inherit
+  trigger-pipeline:
+    runs-on: ubuntu-latest
+    needs: build-images
+    steps:
+      - name: Trigger Pipeline
```

```suggestion
      - name: Trigger CI Images
```

Let's rename this to be more specific